packages/common/src/types/command/command.types.ts

```diff
@@ -10,7 +10,7 @@ import type { CommandV5 } from "./legacy/CommandV5.types";
 
 export type CommandComplete = Required<Omit<CommandLatest, "spokenForm">> &
   Pick<CommandLatest, "spokenForm">;
-export const LATEST_VERSION = 7 as const;
+export const LATEST_VERSION = 7;
```

Comment:
is this not necessary due to some typescript upgrade? seems like this doesn't actually change anything about the type?

Reply:
I'm not sure that something has probably changed wit a typescript update presumably?

Reply:
Nah, this has always been unnecessary. Not possible to mutate a primitive, so always safe to infer the const.